specs/agents/tracing-instrumentation-aws-lambda.md

```diff
@@ -103,9 +103,9 @@ Field | Value | Description | Source
 `context.service.origin.name` | e.g. `gdnrpwmtsb...amazonaws.com` | The full domain name of the API Gateway. | `event.requestContext.domainName`
 `context.service.origin.id` | e.g. `gy415nu...` | `event.requestContext.apiId` |
 `context.service.origin.version` | e.g. `1.0` | `1.0` for API Gateway V1, `2.0` for API Gateway V2. | `event.version` (or `1.0` if that field is not present)
-`context.cloud.origin.service.name` | `api gateway` | Fix value for API gateway. | -
+`context.cloud.origin.service.name` | `api gateway` or `lambda url` | Constant value. | Detect lambda URLs by searching for `.lambda-url.` in the `event.requestContext.domainName`. Otherwise assume API Gateway.
```

Comment:
In the example document there is no `event.requestContext.domainName` field. How about just checking if `event.requestContext.elb` is not null.